spec/classes/server_spec.rb

```diff
@@ -3,7 +3,6 @@
 shared_examples_for "all puppet master types" do
   it { should compile.with_all_deps }
   it { should contain_class('puppet::server') }
-  it { should contain_class('puppet::package') }
```

Comment:
this test should probably be replaced with a test for package inclusion logic rather than removed outright; we should be able to verify that if mange_package etc is set then there's a package resource in the catalog.